addon/templates/components/common/bs-form/element/errors.hbs

```diff
@@ -1,3 +1,11 @@
 {{#if show}}
-  <div class={{feedbackClass}}>{{messages.firstObject}}</div>
+  {{#if showMultipleErrors}}
+    <div class="{{feedbackClass}} pre-scrollable">
```

Comment:
What is `pre-scrollable` here? This one described here: https://getbootstrap.com/docs/4.0/content/code/#code-blocks?

If so, it should be a class, right? And I am not sure we should take the decision to apply that on behalf of the user. Maybe add some own class like `multiple-errors` or so, so the user can apply its own styling he chooses?

Reply:
I think it is much better to have a `scrollable` container by default, otherwise, the page height could increased by several orders of magnitude.
[`pre-scrollable`](https://github.com/twbs/bootstrap/blob/34cd2038d2b16f305919e1fc36ff45913e3d1dfd/dist/css/bootstrap.css#L564-L567) is very handy and uses `max-height` which helps to mimics the `show me only one error` use case.